src/include/cell.h

```diff
@@ -205,7 +204,7 @@ struct __wt_cell_unpack_addr {
 
     WT_PAGE_DELETED page_del; /* Fast-truncate information */
 
-    WT_PAGE_STAT ps; /* Page information including row and byte counts */
+    WT_PAGE_STAT ps; /* Page information including record and user byte counts */
```

Comment:
I don't think we should add this to the in memory data structure. My understanding is that we can unpack page stats from the cell when we actually needs it. No need to keep them around in memory.

Same for __wt_cell_unpack_kv.

Reply:
@quchenhao I understand your intent but I still do think these variables would be useful to keep around for the cell packing logic. If we didn't have the `ps` variable inside, we'd need to pass in a `ps` every time we call `__wt_rec_cell_build_addr`.

Reply:
Yes, I think that's ok. The issue for putting it there is that it gives the programmer a false impression that this value is there to be used anytime. But that is not the case.

Reply:
Removed here: [8754ca1](https://github.com/wiredtiger/wiredtiger/pull/8792/commits/8754ca1e57dd67cd35eaf554bac288d8e7a23d67).